src/App.js

```diff
@@ -1,17 +1,68 @@
 import './App.css';
-import React from 'react';
-
-function App() {
-  // fetches JSON data passed in by flask.render_template and loaded
-  // in public/index.html in the script with id "data"
-  // const args = JSON.parse(document.getElementById("data").text);
-
-  // This main page will be further designed with react components
-  // so the user can interact with the main app page. Some Data will also
-  // be present for viewing when user lands on this page.
-  return (
-    <h1> ! Welcome To Main App Page User !</h1>
-  );
-}
-
-export default App;
+import React from "react";
+import { Dropdown, Option } from "./Dropdown";
+
+/*NOTE:
+  Unaware how to connect fdc.py API calls to search bar function.
+  I will need help trying to figure out how to connectthe search bar 
+  and USDA API to return user requested results. As it stands for an 
+  MVP we have good functionality for the main app page, and we can either
+  add more functionality than what we have now or save some features to add
+  into the app for Sprint 2.
+
+*/
+export default function App() {
+
+  //Will handle the diet components
+  const [optionValue, setOptionValue] = useState("");
+  const handleSelect = (e) => {
+    console.log(e.target.value);
+    setOptionValue(e.target.value);
+  };
+
+  //Will handle the search function
+    const form = useRef(null)
+
+    const submit = e => {
+      e.preventDefault()
+      const data = new FormData(form.current)
+      fetch('https://api.nal.usda.gov/fdc/v1/' + query, { method: 'POST', body: data })
+        .then(res => res.json())
+        .then(data => console.log(data))
+    }
+    return (
+      <div>
+
+        {/* This react component basically gives the app functionality
+        to show users the best overall diets they can partake in.
+        The component is basically a dropdown menu that the user can
+        select the exact diet and click an interactive button to find
+        out more about the diet of interest
+    */}
+
+        <h1>Find The Best Diet !</h1>
+        <Dropdown
+          buttonText="Search"
+          onChange={handleSelect}
+          action="https://health.usnews.com/best-diet/best-diets-overall"
```

Comment:
Will this `action` be modified to use the link that corresponds with the selected option?

Reply:
Not sure how to do this in particular because the base URL for the website shows different pathways to specific diets that can help certain medical ailments. I can attempt to modify the link to correspond with the select action if needed.

Reply:
You can use the `onChange` action to modify the action based on the selected option. It might help to have an array of links in the same order as the dropdown menu options. Select menus have a `selectedIndex` integer field that should make that easier.

Reply:
In Sprint 2 I might look to do this instead and just keep this simple for the MVP.